runtime/gear/src/lib.rs

```diff
@@ -89,12 +88,19 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
     impl_name: create_runtime_str!("gear-node"),
     apis: RUNTIME_API_VERSIONS,
     authoring_version: 1,
-    spec_version: 1650,
+    spec_version: 1660,
```

Comment:
maybe reset spec version due to update to V3?

Reply:
better reset it in PR with json spec and such